frontend/src/styles/utilities.stories.tsx

```diff
@@ -157,3 +157,88 @@ export const TextWeight = (): JSX.Element => {
         </>
     )
 }
+
+export const Widths = (): JSX.Element => {
+    return (
+        <div className="flex flex-col space-y-2">
+            <div className="w-1/5 border rounded text-center">w-1/5</div>
+            <div className="w-1/3 border rounded text-center">w-1/3</div>
+            <div className="w-2/5 border rounded text-center">w-2/5</div>
+            <div className="w-1/2 border rounded text-center">w-1/2</div>
+            <div className="w-3/5 border rounded text-center">w-3/5</div>
+            <div className="w-2/3 border rounded text-center">w-2/3</div>
+            <div className="w-4/5 border rounded text-center">w-4/5</div>
+            <div className="w-full border rounded text-center">w-full</div>
+        </div>
+    )
+}
+
+export const AbsolutePositioning = (): JSX.Element => {
```

Comment:
It seems the stories for `inset` don't really show it off, because the all the blue outlines have `h-20 w-20`, making `inset-0` look the same as `left-0 top-0`